packages/support/README.md

```diff
@@ -0,0 +1,38 @@
+# `support`
+
+> TODO: description
+
+## Usage
+
+```ts
+import { SupportServer } from '@highoutput-library/support';
+
+const app = express();
+
+const supportServer = new SupportServer({
+  notionDatabaseId: faker.git.commitSha(),
+  notionAccessToken: faker.git.commitSha(),
+  prefix: '/support',
+  project: 'HER',
+});
+
+app.use(supportServer.expressMiddleware());
+
+app.listen(8080);
+```
+
+`Add support message to notion`
+
+```json
+// POST localhost:8080/support/messages
+
+{
+  "customer": "alres arena",
```

Comment:
hmm, I guess we should just put the `customerName` inside `details`
```
{
  emailAddress: string;
  message: string;
  details?: Record<string, unknown>
}
```

Reply:
fixed